recipe/build_dali_tf.sh

```diff
@@ -43,4 +43,4 @@ for i in "${TF_CFLAGS[@]}";do
 done
 
 
-$COMPILER -std=c++11 -O2 -shared -fPIC ${SRCS} -o ${LIB_NAME} ${INCL_DIRS} -DNDEBUG ${TF_CFLAGS[@]} ${TF_LFLAGS[@]} ${DALI_CFLAGS[@]} ${DALI_LFLAGS[@]}
```

Comment:
I think this essentially changes from c++11 to c++17 (the default in the conda toolchain.) Is that what we want here? I think there were problems in the past due to c++17.

Reply:
That was on purpose, since I was trying to use $CXXFLAGS to get the proper include file paths and $CXXFLAGS had already set std to c++17.

This builds fine, do you expect there to be some sort of run time issue?

Reply:
Even upstream is still using c++11 here -> https://github.com/NVIDIA/DALI/blob/v0.25.0/dali_tf_plugin/build_dali_tf.sh
We can try the tests from here https://github.com/NVIDIA/DALI/blob/v0.25.0/dali/test/python/test_dali_tf_plugin_run.py to ensure it works.